<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>731</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lieu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Column1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Column2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Column3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Update">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Delete">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Read">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SortDateAsc">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Date ↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SortDateDesc">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Date ↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ExportPDF">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Export to pdf</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Score">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TypeMatch">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Spectateurs">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>310</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type match</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>350</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Spectatures</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MatchStats">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(95, 161, 9);</string>
    </property>
    <property name="text">
     <string>Statistics</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Calendar">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(95, 161, 9);</string>
    </property>
    <property name="text">
     <string>Calendar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SearchTypeMatch">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche (type de match)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-260</x>
      <y>-260</y>
      <width>1381</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Desktop/aa.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Simulate">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation</string>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>tableView</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_ID_2</zorder>
   <zorder>lineEdit_Column1</zorder>
   <zorder>lineEdit_Column2</zorder>
   <zorder>lineEdit_Column3</zorder>
   <zorder>pushButton_Create</zorder>
   <zorder>pushButton_Update</zorder>
   <zorder>pushButton_Delete</zorder>
   <zorder>pushButton_Read</zorder>
   <zorder>pushButton_SortDateAsc</zorder>
   <zorder>pushButton_SortDateDesc</zorder>
   <zorder>pushButton_ExportPDF</zorder>
   <zorder>lineEdit_Score</zorder>
   <zorder>lineEdit_TypeMatch</zorder>
   <zorder>lineEdit_Spectateurs</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>pushButton_MatchStats</zorder>
   <zorder>pushButton_Calendar</zorder>
   <zorder>lineEdit_SearchTypeMatch</zorder>
   <zorder>label_8</zorder>
   <zorder>pushButton_Simulate</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
